README.adoc

```diff
@@ -68,3 +77,39 @@ There are a few edge cases where we might want to work only on the current branc
 ** Create a feature branch from `dev`, to be merged into `dev` when appropriate.
 * When a new version is ready to published, the `5.x` branch will get a git tag, named with the exact version (for example, **5.1.0**), signifying that this point-in-time marks the completion of the work for that minor release.
 * Updates merged into the `dev` branch for the next release are cherry-picked into the `5.x` branch.
+
+[[documenting-changes]]
+== Documenting changes to Cypher
+
+New, updated, deprecated, and removed features must be recorded on the xref:deprecations-additions-removals-compatibility.adoc[] page.
+
+New and deprecated features should also be marked with a label:
+
+* If the impacted feature has its own header in the Cypher Manual, use the following:
+
+[role=label--new-5.x]
+\== Header
+
+[role=label--deprecated]
+\== Header
+
+* If the impacted feature is documented within a table (such as a return column in a `SHOW` command), use the following:
+
+`featureX` label:new[Introduced in 5.x] +
+`featureY` label:deprecated[]
+
+Labels can be difficult to apply to updated features.
+In these cases, it is often preferable to note the change in a sentence.
+For example: "As of Neo4j 5.x, featureX supports ..."
+
+Removed features should be removed from the Cypher Manual.
+
+[[cypher-gql]]
+== Cypher and GQL
+
+When documenting a new Cypher feature, its relationship with GQL must be considered:
+
+* If the feature is part of GQL's mandatory features, it should be recorded on the page xref:modules/ROOT/pages/appendix/gql-conformance/supported-mandatory.adoc[].
+* If the feature is part of GQL's optional features, it should be recorded on the page xref:modules/ROOT/pages/appendix/gql-conformance/supported-optional.adoc[].
+* if the feature adds functionality for which there exists an analogous GQL equivalent, it should be recorded on the page xref:modules/ROOT/pages/appendix/gql-conformance/analogous-cypher.adoc[].
```

Comment:
"analogous GQL equivalent" - is it analogous or equivalent? their use might be synonymous in this context, but then we only need one :thinking: